scss/bootstrap/overrides/style/_forms.scss

```diff
@@ -75,6 +89,36 @@ label {
   }
 }
 
+// Style new group of type radio
+.form-inline {
+
+  &.form-inline-tabs .form-group {
```

Comment:
since you don't need to use `.form-inline` in the selector anymore, you have 3 levels

Anyway it is a matter of consistent, it's easier to scan and read the file if you know that selectors are always nested, instead of having a mix of nested and inline